src/seedu/addressbook/common/Messages.java

```diff
@@ -11,6 +11,7 @@
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSON_NOT_IN_ADDRESSBOOK = "Person could not be found in address book";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_PERSONS_SORTED_OVERVIEW = "%1$d persons sorted and listed!";
```

Comment:
Java coding standards: White space within a statement, are done in the with proper spacing between words and characters